Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -142,7 +147,7 @@ public void SetPlayerInput(InputManager player)
         localGunHolderX = gunHolder.transform.localPosition.x;
         localGunHolderHeight = gunHolder.transform.localPosition.y;
         playerCamera = inputManager.GetComponent<Camera>();
-        startingFov = playerCamera.fieldOfView;
+        startingFov = StaticInfo.Singleton.CameraFov;
```

Comment:
This should be in a different place than the static _info_, but that can be saved for later

Reply:
Yep, I think a static settings component could be a good home for that, we'll hook it up to #383 👍